src/Data/HashMap.purs

```diff
@@ -407,3 +409,8 @@ nubHash :: forall a. Hashable a => Array a -> Array a
 nubHash = runFn4 nubHashPurs Nothing Just (==) hash
 
 foreign import nubHashPurs :: forall a. Fn4 (forall x. Maybe x) (forall x. x -> Maybe x) (a -> a -> Boolean) (a -> Int) (Array a -> Array a)
+
+foreign import anyPurs :: forall k v. (v -> Boolean) -> HashMap k v -> Boolean
+
+any :: forall k v. (v -> Boolean) -> HashMap k v -> Boolean
```

Comment:
Please add some docs. Doesn't have to be much. Just referring to Foldable's any should be enough. Maybe an example if you're feeling inspired.